### PR TITLE
My home: Remove subheadings

### PR DESCRIPTION
**Pull request by lcollette**
#### Proposed changes

Removing the subheadings on My home to decrease the visual clutter on the screen.

**Before:**
Screen Shot 2020-11-17 at 1 05 40 PM

**After:**

Screen Shot 2020-11-17 at 1 05 20 PM

#### Testing instructions

- Visit My home as a new user, as a returning user, and on each of our plan types.
- Notice the lack of subheadings.
- Does the screen flow? Is any spacing off?

### CODE REVIEW

**Review by sixhours — DISMISSED** [review]
This does look so much cleaner!

I noticed the bottom spacing between the main card area and the secondary/tertiary cards is off:

Screen Shot 2020-11-17 at 4 41 42 PM

**Comment by krymson24** [comment]
@lcollette such a subtle change, but I love it!

Alignment fixed:
Screen Shot 2020-11-18 at 1 25 06 AM

The rest:
Screen Shot 2020-11-18 at 1 26 10 AM

**Review by krymson24 — DISMISSED** [review]

File: client/my-sites/customer-home/cards/features/stats/index.jsx
Comment (krymson24) [inline client/my-sites/customer-home/cards/features/stats/index.jsx:87]: Unnecessary space? Not blocking though

**Comment by rickybanister** [comment]
So much better.

There are some other minor spacing issues, but unrelated to this change:

The height of the stats header area that says 'View' is not as tall as the quick links header. 
The padding in the main card is larger than the rest of the cards, but not enough to feel entirely intentional.

**Comment by kwight** [comment]
@lcollette Does this need anything else? I've dismissed the stale reviews and approvals so we're fresh again – let us know in Serenity if you need any getting it done 👍

**Comment by sirbrillig** [comment]
I... think that the rebase might have gotten confused 😅

**Comment by lcollette** [comment]
@sixhours Might I get a review on this before I merge?

**Review by sixhours — APPROVED** [review]
Looks good! 🚢

**Pull request merged**
